### PR TITLE
Bing Ads doc update

### PR DESCRIPTION
**Pull request by ishachopra0401**
## What do these changes do?

Property mappings update in track call

### CODE REVIEW

File: docs/destinations/streaming-destinations/bingads.mdx [hunk 1]
```diff
@@ -42,11 +42,23 @@ rudderanalytics.page()
...
 
-RudderStack maps the `properties.category` and `properties.currency` fields to the corresponding category and currency fields of Bing Ads, respectively. RudderStack also maps `properties.total` and `properties.revenue` or `properties.value` to the variable `revenue`.
+### Supported mappings
 
-To use this call, only the event name is required - all other properties are optional.
+The following table details the mappings between RudderStack and Bing Ads properties:
+
+| RudderStack property | Bing Ads property |
+| :-----| :--------| 
+| `message.event` <br/> <span style="color: #4D4DFF;font-size:12px;">Required</span> | `event` |
```
Comment (sandeepdsvs) [inline docs/destinations/streaming-destinations/bingads.mdx:53]:
```suggestion
| `message.event` <br/> <span style="color: #4D4DFF;font-size:12px;">Required</span> | `event_label` | 
```

**Review by sandeepdsvs — APPROVED** [review]

**Review by ItsSudip — APPROVED** [review]

File: docs/destinations/streaming-destinations/bingads.mdx [hunk 2]
```diff
@@ -42,11 +42,23 @@ rudderanalytics.page()
...
+| RudderStack property | Bing Ads property |
+| :-----| :--------| 
+| `message.event` <br/> <span style="color: #4D4DFF;font-size:12px;">Required</span> | `event_label` | 
+| `properties.category` | `category` | 
+| `properties.currency`  | `currency` |
+| `properties.eventValue` | `event_value` | 
+| `properties.total` <br/>  `properties.revenue` <br/>  `properties.value` | `revenue` | 
+
+<div class="infoBlock">
+<code class="inline-code">properties.total</code> is given the highest priority followed by <code class="inline-code">properties.revenue</code> > <code class="inline-code">properties.value</code>.
```
Comment (ameypv-rudder) [inline docs/destinations/streaming-destinations/bingads.mdx:60]:
```suggestion
<code class="inline-code">properties.total</code> is given the highest priority followed by <code class="inline-code">properties.revenue</code> and <code class="inline-code">properties.value</code>.
```

**Review by ameypv-rudder — APPROVED** [review]
LGTM - just some minor tweaks needed.

File: docs/destinations/streaming-destinations/bingads.mdx (hunk 2 above)
Comment (ishachopra0401) [reply docs/destinations/streaming-destinations/bingads.mdx:60]: Actually, its like properties.total > properties.revenue > properties.value

If we mention 'and', it kind of feels like same priority for the rest of two, which is not the case. Hence, the > sign. LMK your thoughts

Comment (ameypv-rudder) [reply docs/destinations/streaming-destinations/bingads.mdx:60]: How about we rephrase this to:

"RudderStack prioritizes `properties.total` over `properties.revenue` followed by `properties.total`, before mapping it to the Bing Ads' `revenue` property."

My concern is in the original sentence, only one `>` is not conveying the priority clearly.
